src/App/KickOff.php

```diff
@@ -60,8 +74,9 @@ protected function getSections() {
     /**
      * @param Section $section
      */
-    protected function handleSection(Section $section) {
-        $url = $section->getTargetUrl();
+    protected function handleSection(Section $section, $targetUrl = null) {
```

Comment:
We can keep that as you developed it. I am not 100% satisfied, but since I can't think of a better approach, it's ok :-)